tests/integration/test_attach_backup_from_s3_plain/test.py

```diff
@@ -0,0 +1,40 @@
+# pylint: disable=global-statement
+# pylint: disable=line-too-long
+
+import pytest
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance(
+    "node",
+    main_configs=["configs/disk_s3.xml"],
+    with_minio=True,
+)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def start_cluster():
+    try:
+        cluster.start()
+        yield
+    finally:
+        cluster.shutdown()
+
+
+def test_attach_backup():
+    node.query(
+        f"""
+    -- BACKUP writes Ordinary like structure
+    set allow_deprecated_database_ordinary=1;
+    create database ordinary engine=Ordinary;
+
+    create table ordinary.test_backup_attach engine=MergeTree() order by tuple() as select * from numbers(100);
+    -- NOTE: name of backup ("backup") is significant.
+    backup table ordinary.test_backup_attach TO Disk('backup_disk_s3_plain', 'backup');
+
+    drop table ordinary.test_backup_attach;
+    attach table ordinary.test_backup_attach (number UInt64) engine=MergeTree() order by tuple() settings storage_policy='attach_policy_s3_plain';
```

Comment:
And just a note, this will not work for the case with multiple parts, due to files deduplication, but you can workaround this with the following code (but note, that it touches the backup itself, so please, be aware and do not use it on real backups):

```shell
# NOTE: here I use gsutil over s3cmd, since I tested with GCP
gsutil cat gs://path/to/backup_name/.backup > .backup
# NOTE: compose is required to overcome the following issue:
# BadRequestException: 400 Rewriting objects created via Multipart Upload is not implemented yet. As a workaround, you can use compose to overwrite the object (by specifying path with the problem as both the source and output of compose) prior to rewrite.                                               
cat .backup | xq -r '.config.contents.file[] | select(.data_file != null) | [.name, .data_file] | @tsv' | awk -vroot=gs://path/to/backup_name/ '{printf("gsutil cp %s %s || { gsutil compose %s && gsutil cp %s %s; }\n", root $2, root $1, root $2, root $2, root $1)}' > /tmp/backup-data_file-restore.sh
time bash -x backup-data_file-restore.sh |& ts | tee backup-data_file-restore.log
```